docs/developer-guide/base-maps/using-with-arcgis.md

```diff
@@ -0,0 +1,9 @@
+# Using with ArcGIS
+
+Starting with v8.1, deck.gl has support for ArcGIS with the [@deck.gl/arcgis](/docs/api-reference/arcgis/overview.md) module.
+
```

Comment:
Could be nice to have a little table with features for each basemap? A bit like loaders in loaders.gl: https://loaders.gl/modules/csv/docs/api-reference/csv-loader

The table could have rows indicatinh whether this basemap supports overlay and/or interleaved, whether it is 2D/3D etc. link to docs etc.

Reply:
done